odxtools/cli/find.py

```diff
@@ -120,6 +121,14 @@ def add_subparser(subparsers: "argparse._SubParsersAction") -> None:
         help="Relax output formatting rules (allow unknown bitlengths for ascii representation)",
     )
 
+    parser.add_argument(
+        "-od",
+        "--object-details",
+        action="store_true",
+        required=False,
+        help="Print all object details instead of just the name",
```

Comment:
this new argument seems unrelated to the compare function, so it should be moved to its own PR along with any other code that was added for it

Reply:
The _compare_ function uses the _print_service_parameters_ function to display information about the service parameters. The _find_ function uses the _print_diagnostic_service_ function to display diagnostic service details. These util-functions were partially identical in their functionality, so the _print_diagnostic_service_ function has been adapted and now calls the _print_service_parameters_ function if necessary. As the _print_service_parameters_ function has an argument specifying whether the entire diagnostic object should be displayed or not, the _find_ function needed a new argument.

Reply:
The split this PR in two then:
* One PR to add print_objects to print_diagnostic_service
* One PR to add compare tool, and use the print_diagnostic_service

Any other way of splitting this PR is also acceptable, but the PR as is with +1400 changes, is too big for review

See https://smartbear.com/learn/code-review/best-practices-for-peer-code-review/

Reply:
Another PR for functionalities unrelated to compare function was created: #250